### PR TITLE
Better dynamic documentation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Review by ochriste — APPROVED** [review]

File: doc/developer/webservices.rst [hunk 1]
```diff
@@ -420,6 +420,7 @@ Parameters
 * ``partitionlimit``: The maximum number of results per layer (optional).
 * ``lang``: The language used (optional).
 * ``interface``: The interface used (optional).
+* ``ranksystem``: Can me set to ``ts_rank_cd`` to use the ``ts_rank_cd`` rank system instead of ``similarity``.
```
Comment (jwkaltz) [inline doc/developer/webservices.rst:423]: me -> be

File: doc/integrator/ngeo.rst.tmpl [hunk 2]
```diff
@@ -115,7 +115,7 @@ The sub section is the interface name, and after that we have:
     * ``name``: Name of the route whose URL we want to have.
     * ``kw``: Keyword arguments to supply for dynamic path elements in the route definition.
     * ``elements``: Additional positional path segments to append to the URL after it is generated.
-    * ``params``: Query string parameters to append to the URL.
+    * ``params``: Query string parameters to append to the URL (available parameters of ours services: :ref:`developer_webservices`).
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:118]: ours -> our

File: doc/integrator/ngeo.rst.tmpl [hunk 3]
```diff
@@ -124,7 +124,14 @@ The sub section is the interface name, and after that we have:
 
 * ``lang_urls_suffix`` suffix used in l10n URL, see: :ref:`integrator_multi_organization`.
 
-The dynamic values names are: ``interface``, ``cache_version``, ``lang_urls``, ``fulltextsearch_groups``.
+The dynamic values names are:
+
+* ``interface``: The interface name given in parameters.
+* ``cache_version``: The version of the cache.
+* ``two_factor``: Tow factor authentication status from `vars.authentication.two_factor`.
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:131]: Tow -> Two

File: doc/integrator/ngeo.rst.tmpl [hunk 4]
```diff
@@ -124,7 +124,14 @@ The sub section is the interface name, and after that we have:
...
+* ``lang_urls``: The language URL for AngularJS dynamic locale.
+* ``i18next_configuration``: The i18next configuration from `vars.i18next.backend` with dynamically calculated `loadPath`,
+* ``fulltextsearch_groups``: The search groups from the `layer_name` of Full-Text Search table.
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:134]: of -> of the

File: doc/integrator/ngeo.rst.tmpl [hunk 5]
```diff
@@ -124,7 +124,14 @@ The sub section is the interface name, and after that we have:
 
 * ``lang_urls_suffix`` suffix used in l10n URL, see: :ref:`integrator_multi_organization`.
 
-The dynamic values names are: ``interface``, ``cache_version``, ``lang_urls``, ``fulltextsearch_groups``.
+The dynamic values names are:
+
+* ``interface``: The interface name given in parameters.
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:129]: I don't understand the "given in parameters"

Comment (sbrunner) [reply doc/integrator/ngeo.rst.tmpl:129]: => 'given in parameters of the dynamic view' ...

Comment (sbrunner) [reply doc/integrator/ngeo.rst.tmpl]: Is it ok @jwkaltz ?

Comment (jwkaltz) [reply doc/integrator/ngeo.rst.tmpl]: Ah, yes. But "given in the parameters" instead of "given in parameters"

**Review by jwkaltz — APPROVED** [review]

File: doc/integrator/ngeo.rst.tmpl (hunk 5 above)
Comment (sbrunner) [reply doc/integrator/ngeo.rst.tmpl]: Thanks :-)

**Pull request merged**